contracts/fee_currencies.go

```diff
@@ -29,6 +36,21 @@ func init() {
 	}
 }
 
+func getDirectoryAddress(chainId *big.Int) common.Address {
+	if chainId == nil {
+		return addresses.FeeCurrencyDirectoryAddress
+	}
```

Comment:
Why do we need this?

Reply:
In some tests the `ChainId` is `nil`. This is also why we need this check in other parts of the code:
https://github.com/celo-org/op-geth/blob/fa63745dd4b300c27de32ee8c7695b70067efc9f/core/vm/celo_contracts.go#L51

Reply:
I'm not really happy with this, but I certainly don't want to fix all the tests that don't have a chainId. Maybe add a short comment that this is only supposed to happen in tests?